src/fr/charly/javaTutorial/Player.java

```diff
@@ -0,0 +1,44 @@
+package fr.charly.javaTutorial;
+
+public class Player {
+
+	private String name;
+	private double health;
+	private double attack;
+
+	public Player(String name, double attack, double health) {
+		this.name = name;
+		this.attack = attack;
+		this.health = health;
+	}
+
+	public String getName() {
+		return name;
+	}
+
+	public void Damage(double damage) {
```

Comment:
This is a simple method. It should begin with a small letter. Only the constructors can begin with capital letters.